gnovm/pkg/gnolang/preprocess.go

```diff
@@ -2549,28 +2569,43 @@ func addHeapCapture(dbn BlockNode, fle *FuncLitExpr, name Name) uint16 {
 	ne := NameExpr{
 		Path: vp,
 		Name: name,
-		Type: NameExprTypeHeapClosure,
+		Type: NameExprTypeHeapClosure, // XXX, this is actually not used, remove?
```

Comment:
right it's not actually used, but let's leave it because it is a special kind of use of a name, to copy it to func value captures. It just means it was captured by a closure, for now.